### PR TITLE
Support emulator in migrations

### PR DESCRIPTION
**Pull request by SupunS**
Required for https://github.com/onflow/cadence/issues/3063

### CODE REVIEW

File: cmd/util/ledger/migrations/change_contract_code_migration.go [hunk 1]
```diff
@@ -331,16 +330,30 @@ func SystemContractChanges(chainID flow.ChainID) []SystemContractChange {
...
+				)),
+			},
+
+			// EVM related contracts
+			NewSystemContractChange(
+				systemContracts.EVMContract,
+				evm.ContractCode(
+					systemContracts.FlowToken.Address,
+					true,
+				),
```
Comment (SupunS) [inline cmd/util/ledger/migrations/change_contract_code_migration.go:351]: maybe this is working now after https://github.com/onflow/cadence-tools/pull/280

File: cmd/util/ledger/migrations/change_contract_code_migration.go [hunk 2]
```diff
@@ -350,3 +363,20 @@ func mustHexAddress(hexAddress string) common.Address {
...
+	return []*ledger.Payload{
+		// `Burner` contract
+		ledger.NewPayload(
+			convert.RegisterIDToLedgerKey(
+				flow.ContractRegisterID(serviceAccount, "Burner"),
+			),
+			ftContracts.Burner(),
+		),
+	}
+}
```
Comment (SupunS) [inline cmd/util/ledger/migrations/change_contract_code_migration.go:382]: This is only used for test at the moment. We would need to properly 'deploy' this contract somehow

File: cmd/util/ledger/migrations/change_contract_code_migration.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (turbolent) [reply cmd/util/ledger/migrations/change_contract_code_migration.go:351]: No, a reduced version of the full contract is going to get deployed during the "current" HCU

**Review by turbolent — APPROVED** [review]
Nice!

Let's get the new `Burner` contract deployed in a separate PR

**Pull request merged**